app/pages/project/instances/AutoRestartCard.tsx

```diff
@@ -0,0 +1,165 @@
+        bootDisk: instance.bootDiskId,
+        autoRestartPolicy: match(values.autoRestartPolicy)
+          .with('default', () => undefined)
+          .with('never', () => 'never' as const)
+          .with('best_effort', () => 'best_effort' as const)
+          .exhaustive(),
+      },
+    })
+  })
+
+  return (
+    <form onSubmit={onSubmit}>
+      <CardBlock>
+        <CardBlock.Header
+          title="Auto-restart"
+          description="The auto-restart policy for this instance"
+        />
+        <CardBlock.Body>
+          <ListboxField
+            control={form.control}
+            name="autoRestartPolicy"
+            label="Policy"
+            description="The global default is currently best effort, but this may change in the future."
+            items={restartPolicyItems}
+            required
+            className="max-w-lg"
+          />
+          <FormMeta
+            label="Cooldown expiration"
+            tip="When this instance will next restart (if in a failed state and the policy allows it). If N/A, then either the instance has never been automatically restarted, or the cooldown period has expired."
+          >
+            {instance.autoRestartCooldownExpiration ? (
+              <>
+                {toLocaleDateTimeString(instance.autoRestartCooldownExpiration)}{' '}
+                {instance.runState === 'failed' && instance.autoRestartEnabled && (
+                  <span className="text-tertiary">
+                    (
+                    {instanceAutoRestartingSoon(instance)
+                      ? 'restarting soon'
+                      : formatDistanceToNow(instance.autoRestartCooldownExpiration)}
+                    )
+                  </span>
+                )}
+              </>
+            ) : (
+              <span className="text-tertiary">N/A</span>
+            )}
+          </FormMeta>
+          <FormMeta
+            label="Last auto-restarted"
+            tip="When this instance was last automatically restarted. N/A if never auto-restarted."
+          >
+            {instance.timeLastAutoRestarted ? (
+              toLocaleDateTimeString(instance.timeLastAutoRestarted)
+            ) : (
+              <span className="text-tertiary">N/A</span>
+            )}
+          </FormMeta>
+        </CardBlock.Body>
+        <CardBlock.Footer>
+          <LearnMore href={links.instanceUpdateDocs} text="Auto-Restart" />
+          <Button size="sm" type="submit" disabled={disableSubmit}>
+            Save
+          </Button>
+        </CardBlock.Footer>
+      </CardBlock>
+    </form>
+  )
+}
+
+type FormMetaProps = {
+  label: string
+  tip?: string
+  children: ReactNode
+}
+
+const FormMeta = ({ label, tip, children }: FormMetaProps) => (
+  <div>
+    <div className="mb-2 flex items-center gap-1 border-b pb-2 text-sans-md border-secondary">
+      <div>{label}</div>
+      {tip && <TipIcon>{tip}</TipIcon>}
+    </div>
+    {children}
+  </div>
+)
```

Comment:
Unfortunately git can't tell this is almost entirely a move. The only change is taking off `width="medium"` and putting `max-w-lg` on the `ListboxField`.